pyartcd/pyartcd/pipelines/scan_operator.py

```
@@ -0,0 +1,357 @@
+            self.logger.info(f'[DRY-RUN] Would trigger bundle builds for {len(nvrs)} operators: {", ".join(nvrs)}')
+            return
+
+        try:
+            self.logger.info(f'Triggering bundle builds for {len(nvrs)} operators')
+            jenkins.start_olm_bundle_konflux(
+                build_version=self.version,
+                assembly=self.assembly,
+                operator_nvrs=nvrs,
+                group=self.group,
+            )
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)